modules/omx/omx.c

```diff
@@ -0,0 +1,300 @@
+/**
+ * @file omx.c     Raspberry Pi VideoCoreIV OpenMAX interface
+ *
+ * Copyright (C) 2016 - 2017 Creytiv.com
+ * Copyright (C) 2016 - 2017 Jonathan Sieber
+ */
+
+#include "omx.h"
+
+#include <re/re.h>
+#include <rem/rem.h>
+#include <baresip.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+#include <pthread.h>
+
+/* Avoids a VideoCore header warning about clock_gettime() */
+#include <time.h>
+#include <sys/time.h>
+
+/**
+ * @defgroup omx
+ *
+ * TODO:
+ *  * Proper sync OMX events across threads, instead of busy waiting
+ */
+
+static const int VIDEO_RENDER_PORT = 90;
+
+static int EventHandler(OMX_HANDLETYPE hComponent, void* pAppData,
+	OMX_EVENTTYPE eEvent, OMX_U32 nData1, OMX_U32 nData2,
+	void* pEventData)
+{
+	(void) hComponent;
+	switch (eEvent) {
+		case OMX_EventCmdComplete:
+			printf("omx.EventHandler: Previous command completed\n"
```

Comment:
please dont use `printf` which only prints to stdout. Better to use `info` or `debug` which prints
to the logging system with multiple backends.

(in this case perhaps `debug` is best for low-level details)